lsfg-vk/PKGBUILD

```diff
@@ -3,7 +3,7 @@
 # Contributor: PancakeTAS <???>
 
 pkgname=lsfg-vk
-pkgver=r255.7c0c29c
+pkgver=r263.7113d7d
```

Comment:
Maybe with next update we can do properly the pkgver, so that it shows 1.0.0rxxx